### PR TITLE
Fabo/1169 improve version mismatch err

### PR DESCRIPTION
**Pull request by faboweb**
New message (2nd phrase is in development only):
image

❤️ Thank you!

### CODE REVIEW

File: app/src/main/index.js [hunk 1]
```diff
@@ -739,7 +744,7 @@ async function main() {
     fs.writeFileSync(appVersionPath, pkg.version)
   }
 
-  checkGaiaCompatibility(gaiacliVersionPath)
+  await checkGaiaCompatibility(gaiacliVersionPath)
```
Comment (NodeGuy) [inline app/src/main/index.js]: Thanks, I missed that.

Comment (faboweb) [reply app/src/main/index.js]: Finding on that: if you call async functions without await, errors are not passed to the wrapping function. ergo they never surface. :/

Comment (NodeGuy) [reply app/src/main/index.js]: Yes, I know.  :-(

**Comment by fedekunze** [comment]
Closes #1169

**Pull request merged**